lib/features/children/add_member/repository/add_member_repository.dart

```diff
@@ -2,20 +2,21 @@ import 'package:givt_app/core/network/api_service.dart';
 import 'package:givt_app/features/children/add_member/models/member.dart';
 
 mixin AddMemberRepository {
-  Future<void> addMembers(List<Member> members);
+  Future<bool> addMembers(List<Member> members);
 }
 
 class AddMemberRepositoryImpl with AddMemberRepository {
   AddMemberRepositoryImpl(this.apiService);
   final APIService apiService;
 
   @override
-  Future<void> addMembers(List<Member> members) async {
+  Future<bool> addMembers(List<Member> members) async {
     final profilesJsonList = members.map((member) => member.toJson()).toList();
     final body = {
       'profiles': profilesJsonList,
     };
 
-    await apiService.addMember(body);
+    final isSuccess = await apiService.addMember(body);
```

Comment:
same here - why do we need return value? Do we use it somewhere?

Reply:
I used it to make sure the members were added successfully, but as you pointed out that is not necessary. So remoooved